src/windows/file_properties.py

```diff
@@ -171,15 +174,58 @@ def browsePath(self):
 
         starting_folder, filename = os.path.split(self.file.data["path"])
         newFilePath = QFileDialog.getOpenFileName(None, _("Locate media file: %s") % filename, starting_folder)[0]
-        self.txtFilePath.setText(newFilePath)
+
+
+        # don't update if dialog was canceled
+        if newFilePath:
+            # If this path could be an image sequence, get that info and prompt user.
+            seq_info = get_app().window.files_model.get_image_sequence_details(newFilePath)
+            get_app().window.files_model.ignore_image_sequence_paths = []
+
+            # create the proper path for an image sequence
+            if seq_info:
```

Comment:
So only allowing paths that pass `os.path.isfile()` would exclude image sequences.

So I check if the new path *could* be an image sequence (the code for that automatically prompts the user) and if they accept, the below code will format the path correctly

Reply:
Correct, isfile() is too restrictive